policyengine_us/parameters/gov/states/ca/tax/income/credits/earned_income/earned_income_amount.yaml

```diff
@@ -1,25 +1,28 @@
-description: EITC phase-in end 
+description: California EITC phase-in income end 
 metadata:
   rate_unit: currency-USD
   rate_period: year
   threshold_unit: child
   type: single_amount
-  label: CalEITC phased-in income
+  label: California EITC phase-in income end
   reference:
     - title: California Legal code REVENUE AND TAXATION CODE - RTC (2019)
       href: https://leginfo.legislature.ca.gov/faces/codes_displaySection.xhtml?lawCode=RTC&sectionNum=17052
-    - title: CA EITC Form 3514 instructions (2022)
-      href: https://www.ftb.ca.gov/forms/2022/2022-3514-instructions.html
+    - title: CA EITC Form 3514 instructions (various years)
+      href: https://www.ftb.ca.gov/forms/202X/202X-3514-instructions.html
```

Comment:
Could you list them separately? We'd like them to be valid URLs (at some point, we'll link them from the web app).

Reply:
Done